cuegui/cuegui/Utils.py

```diff
@@ -141,14 +141,14 @@ def isRootGroup(object):
     """Returns true if the object is a root, false if not
     @return: If the object is a root group
     @rtype:  bool"""
-    return object.__class__.__name__ in ["NestedGroup", "Group"] and not object.hasParent()
+    return object.__class__.__name__ in ["NestedGroup", "Group"] and object.isRootGroup()
```

Comment:
Can these checks use `isinstance(object, Group)` instead of the name check?

Reply:
Done. In one case below I had to use `type` to avoid the inheritance.